### ISSUE TITLE
Malformed tar files with ResNet test data

### ISSUE THREAD

**Issue by postrational** [onnx/models#309]
There seem to be some malformed tar files in the onnx/models LFS archives:

```
$ wget https://media.githubusercontent.com/media/onnx/models/master/vision/classification/resnet/model/resnet101-v1-7.tar.gz
--2020-05-21 15:48:44--  
Request sent, awaiting response... 200 OK
Length: 167595867 (160M) [application/octet-stream]
Saving to: ‘resnet101-v1-7.tar.gz’

resnet101-v1-7.tar.gz  100%[===========================================>] 159.83M  5.29MB/s    in 24s

$ tar -xzvf resnet101-v1-7.tar
x resnet101v1/
x resnet101v1/resnet101-v1.onnx
x resnet101v1/test_data_set_0/
x resnet101v1/test_data_set_0/._input_0.pb
x resnet101v1/test_data_set_0/._output_0.pb
x resnet101v1/test_data_set_0/input_0.pb
x resnet101v1/test_data_set_0/output_0.pb
tar: Ignoring malformed pax extended attributeIgnoring malformed pax extended attributeIgnoring malformed pax extended attribute
x resnet101v1/test_data_set_1/
x resnet101v1/test_data_set_1/._input_0.pb
x resnet101v1/test_data_set_1/._output_0.pb
x resnet101v1/test_data_set_1/input_0.pb
x resnet101v1/test_data_set_1/output_0.pb
tar: Ignoring malformed pax extended attributeIgnoring malformed pax extended attributeIgnoring malformed pax extended attribute
x resnet101v1/test_data_set_2/
x resnet101v1/test_data_set_2/._input_0.pb
x resnet101v1/test_data_set_2/._output_0.pb
x resnet101v1/test_data_set_2/input_0.pb
x resnet101v1/test_data_set_2/output_0.pb
tar: Damaged tar archive
tar: Error exit delayed from previous errors.
```

It seems that all of these models are affected (most ResNet models):
```
onnx/models/master/vision/classification/resnet/model/resnet101-v1-7.tar.gz
onnx/models/master/vision/classification/resnet/model/resnet101-v2-7.tar.gz
onnx/models/master/vision/classification/resnet/model/resnet152-v1-7.tar.gz
onnx/models/master/vision/classification/resnet/model/resnet152-v2-7.tar.gz
onnx/models/master/vision/classification/resnet/model/resnet18-v2-7.tar.gz
onnx/models/master/vision/classification/resnet/model/resnet34-v1-7.tar.gz
onnx/models/master/vision/classification/resnet/model/resnet34-v2-7.tar.gz
onnx/models/master/vision/classification/resnet/model/resnet50-v1-7.tar.gz
onnx/models/master/vision/classification/resnet/model/resnet50-v2-7.tar.gz
```

**Comment by postrational** [comment]
@vinitra Could you take a look?

**Comment by vinitra** [comment]
Thanks for letting me know! Investigating. I'm having a hard time replicating the results -- it seems like pressing the download button on the tar.gz files you mentioned and extracting with 7zip allows me to extract the files successfully. Will look into the interaction with wget further.

**Comment by vinitra** [comment]
Addressed in #312.

**Comment by vinitra** [comment]
Closing this as fix has been merged.
